resources/views/stores/index.blade.php

```diff
@@ -30,7 +34,8 @@
                         <input type="hidden" name="store_id" value="{{ $detail->id }}">             
                         <button type="submit" class="btn text-danger"><img src="./css/rogo_okiniiri.jpg" alt="お気に入り星" class="rogo"></button>
                 </form>
-            <a href=""><img src="./css/rogo_addreview.jpg" alt="口コミ鉛筆" class="rogo"></a>
+                <a hreg="{{route('stores.create', $detail->id)}}">
```

Comment:
hreg じゃなくて hrefですね！:eyes:
あとurlは今回こんな感じでお願いします！
（作成ページへのpathの話がちょうど昨日でていたのですがslackではto野呂さんになっていたのでリンク貼っておきます！）
https://engineer-class.slack.com/archives/G01GL8E1PGF/p1623688144194900

```suggestion
                <a href="/reviews/create?id={{$detail->id}}">
```